src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -819,68 +812,8 @@ public void run() {
    * @return true if we have any out of date maps
    */
   private static boolean checkForUpdatedMaps() {
-    try {
```

Comment:
We're refactoring map download control and functionality out of this top level class to "MapDownloadController". The goal/idea is to improve the code design by isolating and breaking up the functionality and complexity. GameRunner2 is a bit of a monolith, it should not contain knowledge of how to download maps, it's unnecessary coupling. Overall we are chipping away at the monolith and reducing coupling, and creating new class that will adhere to SRP